client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -142,7 +142,16 @@ function CheckoutSummaryFeaturesList() {
 
 function SupportText( { hasPlanInCart, isJetpackNotAtomic } ) {
 	const translate = useTranslate();
+	const plan = usePlanInCart();
+
 	if ( hasPlanInCart && ! isJetpackNotAtomic ) {
+		if (
+			'personal-bundle' === plan.wpcom_meta?.product_slug ||
+			'personal-bundle-2y' === plan.wpcom_meta?.product_slug
+		) {
+			return <span>{ translate( 'Access unlimited email support' ) }</span>;
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Access to unlimited email support.' )` *ES Score: 9*